dockerfiles/Dockerfile.cuda

```diff
@@ -0,0 +1,59 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime with CUDA, CUDNN integration
+
+# nVidia cuda 10.0 Base Image
+FROM nvidia/cuda:10.0-cudnn7-devel
+
+# Add conda to PATH
+WORKDIR /work/deps
+
+# Dependencies: Ubuntu-core
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends sudo \
+    libopenblas-base \
+    python3 \
```

Comment:
do we need libopenblas, python3 and python3-pip ?

Reply:
removed!